packages/babel-plugin-transform-modules-systemjs/src/index.js

```diff
@@ -391,26 +391,17 @@ export default declare((api, options) => {
                     for (const specifier of specifiers) {
                       const binding = scope.getBinding(specifier.local.name);
                       // hoisted function export
-                      if (
-                        binding &&
-                        t.isFunctionDeclaration(binding.path.node)
-                      ) {
+                      const bindingPath = binding.path;
+                      if (bindingPath.isFunctionDeclaration()) {
                         exportNames.push(specifier.exported.name);
                         exportValues.push(t.cloneNode(specifier.local));
-                      }
-                      // only globals also exported this way
-                      else if (!binding) {
```

Comment:
This branch is removed, because a global export
```js
export { global }
```
have thrown `global` is undefined since #9589. The transformer can safely assume `binding.path` always existed.